packages/components/src/Table/Table.tsx

```diff
@@ -36,8 +36,23 @@ export interface TableProps
     Omit<TableStateProps<object>, 'showSelectionCheckboxes'> {
   variant?: string;
   size?: string;
+
+  /**
+   * Stretch table to fill the container.
+   * @default false
+   */
   stretch?: boolean;
+
+  /**
+   * Make the column sticky to the top of the table.
+   * @default none
```

Comment:
i think this should be default false right?

Reply:
yeah it shouldn't be none, but i think it should be true